### DISCUSSION TITLE
Getting error on uploading multiple images

### DISCUSSION THREAD

**Discussion by tauseedzaman** [filamentphp/filament#6711] [General]
![image](https://github.com/filamentphp/filament/assets/64689921/940585a2-3e64-4f17-bdd3-57f8bf8ee61f)
![image](https://github.com/filamentphp/filament/assets/64689921/1bac66d5-72eb-4edf-a262-b3d25dab8c43)

I am getting this error when I click on submit button

any idea might be appreciated

**Comment by danharrin** [comment]
Missing `array` cast on your model

**Reply by tauseedzaman** [reply]
how can I fix it.
can you provide example code snippet plz

**Reply by pboivin** [reply]
https://laravel.com/docs/10.x/eloquent-mutators#array-and-json-casting
